### DISCUSSION TITLE
The sound on my laptop gets ruined once i launch the game

### DISCUSSION THREAD

**Discussion by AlehSarhajeu** [FakeFishGames/Barotrauma#13682] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Basically, i have bought a new wireless Razer Blackshark V2 Pro (2023) headset several days ago, and when i launched Barotrauma my volume started jumping and being somewhat distorted, sometimes being really loud, often cracking, and even outright disappearing at times. The whole thing affects not only the game, but the whole system entirely, even if i switch game's playback device to speakers. 

### Reproduction steps

Simply as i lunch the game, my sound breaks 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

Doesnt matter, the sound is distorted all the time, even in the menu

### Version

v1.2.7.0 (Winter Update hotfix)

### Which operating system did you encounter this bug on?

Windows 11

**Comment by Jasontti** [comment]
Hello and thanks for your report.

Tried to reproduce this issue with generic bluetooth headset, but after several tries i could not get the issue you described to happen.
I got one case where bluetooth audio profile got downgraded, which could be just the cheap nature of my headset doing tricks.

We've also had number of different BT audio related fixes during this time, so if you could check if the issue still persists.

If it does, you should check what it says as the audio codec before and after the issue happens.
It should be visible in
Bluetooth & other devices
Advanced Settings
More Bluetooth options
Audio tab.
And codec
